### PR TITLE
add cluster-api-release-team github team

### PR DESCRIPTION
**Pull request by ykakarap**
This PR adds a new github team called the `cluster-api-release-team`.

### CODE REVIEW

File: config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml [hunk 1]
```diff
@@ -259,6 +259,21 @@ teams:
...
+    - cpanato
+    - dntosas
+    - furkatgofurov7
+    - hackeramitkumar
+    - nawazkh
+    - oscr
+    - sayantani11
+    - VibhorChinda
+    - ykakarap
+    privary: closed
```
Comment (furkatgofurov7) [inline config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml:276]: nit: 
```suggestion
    privacy: closed
```

**Comment by fabriziopandini** [comment]
+1

**Review by CecileRobertMichon — COMMENTED** [review]
/lgtm

**Comment by sbueringer** [comment]
/lgtm

Let's wait for folks to get into the org then Fabrizio can approve

**Comment by ykakarap** [comment]
Both the members are now members of the kubernetes-sigs org

(I will try a force push to try to retrigger the tests. `/retest` did not work and I am not able see the test pipeline failures - no details button)

**Comment by ykakarap** [comment]
> Both the members are now members of the kubernetes-sigs org
> 
> (I will try a force push to try to retrigger the tests. `/retest` did not work and I am not able see the test pipeline failures - no details button)

Seeing similar failures in other repos too. Looks like an infra issue.

**Pull request merged**